src/Glpi/Form/AccessControl/FormAccessControl.php

```diff
@@ -82,7 +82,7 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0): string
             $count = count($form_access_mananger->getActiveAccessControlsForForm($item));
         }
 
-        return self::createTabEntry(self::getTypeName(), $count);
+        return self::createTabEntry(self::getTypeName($count), $count);
```

Comment:
Tabs are almost always using the plural form.

```suggestion
        return self::createTabEntry(self::getTypeName(Session::getPluralNumber()), $count);
```